src/app/shell/details/workshop-details/workshop-details.component.html

```diff
@@ -17,6 +16,7 @@
           fxLayoutAlign="start center"
         >
           <h1 class="title">{{ workshop.title }}</h1>
+          <div [style.background-color]="isContinued ? '#99CD56' : 'red'" class="chip-color">{{ isContinued ? recruitmentStatusUkr.RecruitmentOngoing : recruitmentStatusUkr.RecruitmentStoped }}</div>
```

Comment:
In that case I would use ngClass for setting needed class and stylize depending on the class in css file. This is the more common practice

Reply:
> In that case I would use ngClass for setting needed class and stylize depending on the class in css file. This is the more common practice

in this case I'm changing only one field in CSS class that's why I'm sure that there is no need to create extra classes

Reply:
fixed